tests/baselines/reference/parserComputedPropertyName16.types

```diff
@@ -3,7 +3,7 @@ enum E {
 >E : E
 
   [e] = 1
->[e] : E.__computed

```

Comment:
similar change here

Reply:
This one was kinda bad previously. We should've never revealed the `__computed` internal name.